### PR TITLE
fix(weave): fix run colors

### PR DESCRIPTION
**Pull request by jwlee64**
Chart colors were broken here https://github.com/wandb/weave/pull/865/files 

Properly unnest tagged list types for the mapeach so that run colors are properly mapped

before
Screenshot 2024-01-30 at 1 41 23 PM

after
Screenshot 2024-01-30 at 1 40 32 PM

### CODE REVIEW

File: weave-js/src/components/Panel2/panellib/libcolors.ts [hunk 1]
```diff
@@ -22,11 +19,14 @@ export const useColorNode = (inputNode: Node): NodeOrVoidNode => {
...
-      !isAssignableTo(
-        inputNode.type,
-        taggedValue(typedDict({run: 'run'}), 'any')
-      )
+      isVoidNode(frame.runColors)
+      // This was added by this pr https://github.com/wandb/weave/pull/865/files
+      // to fix a slow query, it also as a result broke run colors in charts
+      // I am going to comment out to fix run colors but we should take another look at this later
+      // || !isAssignableTo(
+      //   inputNode.type,
```
Comment (jlzhao27) [inline weave-js/src/components/Panel2/panellib/libcolors.ts:27]: out of curiosity, for the case that previously got caught in this check, what was the type of `inputNode`?

Comment (jwlee64) [reply weave-js/src/components/Panel2/panellib/libcolors.ts:27]: for the case I was looking at it was all typedDict({project: project})

**Review by tssweeney — CHANGES_REQUESTED** [review]
Please see slack comment https://weightsandbiases.slack.com/archives/C03BSTEBD7F/p1706737504949139?thread_ts=1706651271.691759&cid=C03BSTEBD7F

**Review by tssweeney — APPROVED** [review]
🔥 ! Sick! Nice work

**Pull request merged**